scripts/functions/requirements/devuan

```diff
@@ -9,7 +9,7 @@ requirements_devuan_define_libgmp()
 
 requirements_devuan_define_libreadline()
 {
-  requirements_check libreadline6-dev
```

Comment:
Looks good in principle, but could you please make it dependent on the system version in a similar way like for example [here](https://github.com/rvm/rvm/blob/master/scripts/functions/requirements/debian#L74) or [here]()

Reply:
I can that, but it's not really necessary, because all Devuan versions contain `libreadline-dev`, see https://pkginfo.devuan.org/cgi-bin/policy-query.html?c=package&q=libreadline-dev
`libreadline6-dev` only existed in Devuan's first version ("jessie"), being a dependency of `libreadline-dev`, i.e.  `libreadline-dev` and `libreadline6-dev` were the same there.

Reply:
Indeed! Thanks for explanation :)